tests/msbuild/Xamarin.MacDev.Tasks.Tests/TaskTests/LocalizationStringTest.cs

```diff
@@ -96,66 +96,6 @@ private string TranslateError (string culture, string errorCode)
 		IList<string> commonIgnoreList = null;
 		static string shortCommonPath = "xamarin-macios/tests/msbuild/Xamarin.MacDev.Tasks.Tests/TaskTests/LocalizationIgnore/common-Translations.ignore";
```

Comment:
This does not look like is longer used, is it?

Reply:
Yes, you are correct, thanks!